src/GovUk.Education.ExploreEducationStatistics.Data.Api.Tests/GovUk.Education.ExploreEducationStatistics.Data.Api.Tests.csproj

```diff
@@ -19,6 +19,7 @@
     </ItemGroup>
 
     <ItemGroup>
+      <ProjectReference Include="..\GovUk.Education.ExploreEducationStatistics.Admin.Tests\GovUk.Education.ExploreEducationStatistics.Admin.Tests.csproj" />
```

Comment:
Why are we including this reference? If we're trying to access some util, it should probably be moved to another test project like `Common.Tests` or `Data.Model.Tests`

Reply:
This reference is for `DbUtils`. 

If I move it to `Common.Tests`, I'll have to add things like `Content.Model` to `Common.Tests`, and that doesn't seem right.

I think the real home for `DbUtils` is `Data.Model.Tests`, but I'll have to add a lot of references, like adding `IdentityServer4` to `Data.Model.Tests`, which maybe means maybe that should be moved to?

I don't see a big downside to just adding the Admin.Tests reference here, as it is just for unit tests, and it's probably going to take a while to move DbUtils to Data.Model.Tests as it is used a lot. So I'm inclined to leave this now and create a ticket for it. But happy to go along with what you'd prefer.

Reply:
Let's leave it for now as it sounds like it'd be a lot of work. You're right that just having to a reference to `Admin.Tests` isn't really causing any harm